### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by glolsh** [Lithane97/Divine-Intervention-Base#2]
1. Immortality trait should have set_immortal_age = 25 or something like that, so chars won't look like withered potatoes. Or maybe make another trait like - eternally young.
2. Building trait should have holdings build speed up
3. Building modifiers mixed up I think. You modifier that speeds up process is red, which is negative, but it's a good thing...like a bonus.
Should look like this (changed icons only)
```
holding_build_speed = {
	icon = county_modifier_development_negative
	holding_build_speed = 0.01
	stacking = yes
} 
holding_build_speed_neg = {
	icon = county_modifier_development_positive
	holding_build_speed = -0.01
	stacking = yes
```
5. Maybe add religion editor or something? 
Also thank you for you work. I tried using other cheat mod, it really sucks. Yours is best

**Comment by Lithane97** [comment]
Yes, I have noticed the immortality issue, I would like to be able to pick and age (like you can for character spawner) and that is your immortality age. 
I'll edit the

**Comment by Lithane97** [comment]
Whoops wrong button.
Yes, I have noticed the immortality issue, I would like to be able to pick and age (like you can for character spawner) and that is your immortality age.
I'll look at the building traits/modifiers.
I plan to add a religion editor at some point, currently fussing with a culture editor, which I've made some good progress with. It's not perfect though.

**Comment by glolsh** [comment]
Thank you. Also, would be good if you make some sort of interaction like - DESTROY title. My holy order guys bugged, I removed lease, but they still appear in army screen. I failed at writing script to destroy their title.

**Comment by Lithane97** [comment]
I plan to make a menu where you will be able to do all kinds of actions with titles, including destroying.

**Comment by glolsh** [comment]
About immortality age.
Here is my info.
I made it so it freezes age at 25. but if trait is add too early - before 25, then child will stop growing visually at like 10 years or so, and you'll need to set actual age to needed one and re-add trait.

**Comment by xDestra** [comment]
So this is where you put the thing that you like to add? alright. I would like to request a simple name editor. y'know, for roleplay purposes :p

**Comment by Lithane97** [comment]
> So this is where you put the thing that you like to add? alright. I would like to request a name editor. y'know, for roleplay purposes :p (although maybe there _already is_ a way to edit the names of everyone, and i have not found it yet)

There should be a character interaction when you right-click a character to change that character's name.
